subsys/bluetooth/controller/hal/nrf5/radio.c

```diff
@@ -60,11 +60,39 @@ void radio_reset(void)
 	     RADIO_POWER_POWER_Msk);
 }
 
-void radio_phy_set(u8_t phy)
+void radio_phy_set(u8_t phy, u8_t flags)
 {
-	NRF_RADIO->MODE =
-	    (((phy) ? (u32_t)phy : RADIO_MODE_MODE_Ble_1Mbit) <<
-	     RADIO_MODE_MODE_Pos) & RADIO_MODE_MODE_Msk;
+	u32_t mode;
+
+	switch (phy) {
+	case BIT(0):
```

Comment:
I am thinking if I should be using enumeration values instead, as only one PHY needs to be applied at the radio hal/driver layer. But also, I dont want to have additional processing to translate from BIT number value coming in PDU to enumeration values. 

Will do a little analysis using enum values and compare. Either way I will use the HCI defines (bit number or enum. values)